src/UrqlClient.re

```diff
@@ -108,58 +108,40 @@ external executeMutation:
 external executeSubscription:
   (
     ~client: t,
-    ~subscription: UrqlTypes.graphqlRequest,
-    ~opts: option(UrqlTypes.partialOperationContext)=?,
+    ~subscription: graphqlRequest,
+    ~opts: partialOperationContext=?,
     unit
   ) =>
   Wonka.Types.sourceT('a) =
   "";
 
 [@bs.send]
 external executeRequestOperation:
-  (~client: t, ~operation: UrqlTypes.operation) =>
-  Wonka.Types.sourceT(UrqlTypes.operationResult) =
+  (~client: t, ~operation: operation) => Wonka.Types.sourceT(operationResult) =
   "";
 
 [@bs.send]
-external reexecuteOperation:
-  (~client: t, ~operation: UrqlTypes.operation) => unit =
-  "";
+external reexecuteOperation: (~client: t, ~operation: operation) => unit = "";
 
 [@bs.send]
 external createRequestOperationJs:
   (
     ~client: t,
     ~operationType: string,
-    ~request: UrqlTypes.graphqlRequest,
-    ~opts: option(UrqlTypes.partialOperationContext)=?,
+    ~request: graphqlRequest,
+    ~opts: partialOperationContext=?,
     unit
   ) =>
-  UrqlTypes.operation =
+  operation =
   "createRequestOperation";
 
-let createRequestOperation =
-    (
-      ~client: t,
-      ~operationType: UrqlTypes.operationType,
-      ~request: UrqlTypes.graphqlRequest,
-      ~opts: option(UrqlTypes.partialOperationContext)=?,
-      (),
-    ) => {
-  let type_ = UrqlTypes.operationTypeToJs(operationType);
-  createRequestOperationJs(
-    ~client,
-    ~operationType=type_,
-    ~request,
-    ~opts,
-    (),
-  );
+let createRequestOperation = (~client, ~operationType, ~request, ~opts, ()) => {
```

Comment:
In the previous version `~opts` is an optional argument, making it non-optional is intended or a typo? If intended, you can also delete the last `()`

Reply:
Not intentional, good catch!